charts/infoscale/infoscale-vtas-0.0.1/infoscale-vtas.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: sro.openshift.io/v1beta1
+kind: SpecialResource
+metadata:
+  name: infoscale-vtas
+spec:
+  namespace: infoscale-vtas
+  chart:
+    name: infoscale-vtas
+    version: 0.0.1
+    repository:
+      name: infoscale
+      url: file:///charts/infoscale
+  set:
+    kind: Values
+    apiVersion: sro.openshift.io/v1beta1
+    runArgs:
```

Comment:
Actually we do not need `runArgs` here, it was in the old API where we had it explicitly stated. We can omit it then all values could be accessed `.Values.infoscaleConfigMap` rather then `.Values.runArgs.infoscaleConfigMap`

Reply:
We are using runArgs just as a variable/field name here to aggregate all the configurable parameters required for InfoScale deployment and is referred as a single field/unit inside InfoScale operator as well. In case the name "runArgs" creates confusion, will rewording it suffice ?

Reply:
No this is fine. Lets keep it that way.